packages/forms/docs/10-upgrade-guide.md

```diff
@@ -101,3 +105,44 @@ An easy way to upgrade your code quickly is to find and replace:
 
 - `Closure $get` to `\Filament\Forms\Get $get`
 - `Closure $set` to `\Filament\Forms\Set $set`
+
+#### TextInput numeric masks are replaced with Alpine.js masks
```

Comment:
Not just numeric masks, the entire mask object has been replaced. Please use `` for code snippets.

Reply:
>Please use `` for code snippets.

``` is the proper markdown method for code snippets, and is used throughout the existing docs, so I'm not sure if you meant something else here.

Reply:
I mean `TextInput` here instead of TextInput, for example